tests/behat/features/web/steal/steal.feature

```diff
@@ -0,0 +1,23 @@
+@web
+Feature: I can steal a project from an other user
+
+  Background:
+    Given there are users:
+      | id | name     |
+      | 1  | Andi     |
+      | 2  | Klaus    |
+    And there are projects:
+      | id | name      | description             | owned by |
+      | 1  | project 1 | Klaus ist brave         | Klaus    |
+      | 2  | project 2 | Klaus hat zwei project  | Klaus    |
+
+  Scenario: I want to steal the program form Klaus
+    Given I am logged in as "Andi"
+    When I go to "/app/project/1"
+    And I wait for the page to be loaded
+    And I should see "Klaus"
+    And I click "#steal-button"
+    And I wait for AJAX to finish
+    Then I should be on "/app/project/1"
+    And I wait for the page to be loaded
+    And I should see "Andi"
```

Comment:
#### Issues with this test:

This scenario does not validate if the change is permanent. 
Option a) Visit a different page, go back to the project page. Change should still be there.
Drawback: the change could just be stored in the cache/ls.
Option b) Create a new method that validates the owner of a project in the Database.
Drawback: more effort ;)

Additionally, I would at least check at the beginning of the test that "I should not see "Andi". Since this user is logged in there is a high chance his name is displayed anywhere on the project page.

Since you have created a new directory for the test "steal". This directory has to be registered in the behat suites and also the CI (Github Actions config)


#### Tests missing for:

- not logged in
- your own project